en/docs/references/define-configurable-variables.md

```diff
@@ -0,0 +1,126 @@
+
+    1. Click the edit icon on the last return statement (shown in the image below).
+
+        ![Edit return statement](../assets/img/configurables/edit-return-statement.png){.cInlineImage-half}
+
+        This **Return** pane opens, displaying the existing return statement.
+
+    2. Click **`name`** and then click **Expressions**.
+   
+    3. Under **Arithmetic**, click **Es + Ex**.
+   
+    4. Double-click **`add-expression`**, and enter `"Please email your application to "`.
+   
+    5. Click **Es + Ex** again, and then click **Suggestions**.
+   
+    6. In the list of suggestions, click **email**. Here, you are selecting the configurable variable that you previously added.
+
+    7. Click **Save**.
+   
+        You have completed editing the REST API configuration.
+
+4. To view the Pro-Code view of the REST API, click the icon marked in the image below:
+
+    ![Open pro-code view](../assets/img/configurables/open-code-view.png){.cInlineImage-full}
+
+    You will see the configurable you added in the code view as `configurable string email = ?;`.
+
+5. Commit and push the REST API changes you made to GitHub so that they are also available in the Choreo Console. For instructions, see [Create Your First REST API - Step 1.5: Commit the REST API to GitHub](../tutorials/create-your-first-rest-api.md#step-15-commit-the-rest-api-to-github).
+
+6. Open the browser tab in which you are running the Choreo console and refresh it.
+
+7. Click **Deploy** for your REST API.
+
+8. To deploy the REST API, click **Config & Deploy**. 
+
+    The **Configure & Deploy** pane opens on the left of the page. The **email** field in this pane is the configurable you added to the REST API implementation. Choreo saves the value you enter here in an Azure key vault.
+
+9. In the **email** field, enter any email address.
+
+10. Click **Deploy**.
+
+     Once Choreo completes the deployment, the **Configurables** field indicates that the Rest API has a configured key.
+
+      ![Configurables](../assets/img/configurables/configurables.png){.cInlineImage-full}
+
+      If you want to enter a different value for the configurable, you can click on the edit icon in this field and repeat steps 9-10.
+
+Now you have tried out adding a basic configurable. Proceed to the subsections to learn how to add configurables with default values and define configurable variables as you develop your components.
+
+## Define a configurable with a default value
+
+When you add a configurable to your component in the Web Editor, you can enter a specific value as the configurable value instead of `?` as shown below.
+
+```
+configurable <VARIABLE_TYPE> <VARIABLE_NAME> = <DEFAULT_VALUE>
+```
+e.g., If you want to add a configurable named password of which the value should be in string format and with `secret` as the default value, you can add the configurable statement as follows:
```

Comment:
shall we avoid saying "and with" in the sentence? Perhaps just keep either and or with.